docs/experts/expectations.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+title: "Experts: Expectations"
+---
+
+## Definitions
+
+- **work session**: This should be interpreted as "every time you come online after a period of significant break lasting more than a few hours". There is flexibility to bend this definition but that will depend on the needs of the team you are working with.
+- **via Slack**: This will depend on the team you are working with, however the appropriate channel to feed your updates into will be clearly indicated to you by us.
+
+## Your Responsibilities
+
+### Communication
+
+At the start of each work session, publish a self-stand-up via Slack, to include:
+
+- What you achieved last session
+- What Jira ticket you will be working on this session
+
+Attend the regular, in-person stand-up calls, as agreed with us.
+These are typically held weekly.
+
+You must flag via Slack when you are running low on work.
+We'll then work with you to line up the next series of Jira tickets.
```

Comment:
Jira or Github ticket?

Reply:
I took this from @jamienewcomb's [original document](https://ably.atlassian.net/wiki/spaces/DEL/pages/895090726/Managing+our+freelancers) (internal Confluence page).

I was comfortable leaving this as just "Jira" here:

- This page refers to "GitHub First" elsewhere, and I intend to _also_ publish (move) our [GitHub First](https://ably.atlassian.net/wiki/spaces/SDK/pages/1489174532/GitHub+First) (internal Confluence page) policy to this site soon so that is also open / public.
- From a delivery perspective, which much of this content relates to, Jira is still our primary tool for queuing work in priority order as well as tracking progress.